Assets/Scripts/ItemComponents/PainterProperties.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+namespace ItemComponents
+{
+    [CreateAssetMenu(fileName = "Painter Properties", menuName = "Painting/Painter Properties", order = 0)]
+    public class PainterProperties : ScriptableObject
+    {
+        [SerializeField] private GameObject decalPrefab = null;
+        [SerializeField] private ItemSupply itemSupply = new ItemSupply();
```

Comment:
`private ItemSupply itemSupply = default;` is a good alternative too, but that's up to you.